src/main/java/org/jboss/pnc/api/dto/Gav.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2024 Red Hat, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.jboss.pnc.api.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Builder;
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+/**
+ * GAV identifier of the artifact.
+ */
+@Builder
+@Value
+@Jacksonized
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Gav {
```

Comment:
Maybe you could take inspiration/make it compatible with https://github.com/project-ncl/dependency-analysis/blob/master/reports-model/src/main/java/org/jboss/da/model/rest/GAV.java